app/screens/channel/channel.android.js

```diff
@@ -23,8 +21,7 @@ export default class ChannelAndroid extends ChannelBase {
 
         const channelLoaderStyle = [style.channelLoader, {height}];
         const drawerContent = (
-            <SafeAreaView>
```

Comment:
I saw this last week and thought "this seems odd to me, but I guess there's some reasoning behind it.. maybe to support notched Android devices." 😆